crates/snapbox/Cargo.toml

```diff
@@ -49,6 +49,11 @@ path = ["tempfile", "walkdir", "dunce", "detect-encoding", "filetime"]
 ## Snapshotting of commands
 cmd = ["os_pipe", "wait-timeout"]
 
+## Snapshotting of json
+json = ["structured-data"]
+## Snapshotting of structured data
+structured-data = ["serde_json"]
```

Comment:
Could you rebase and switch this to `dep:serde_json`?

I just bumped MSRV to 1.60 so we don't flood users with all optional dependencies as features